dockerfiles/ocaml/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM golang:1.10-alpine
+WORKDIR /go/src/github.com/sourcegraph/lsp-adapter
+COPY . .
+RUN CGO_ENABLED=0 GOBIN=/usr/local/bin go install github.com/sourcegraph/lsp-adapter
+
+# 👀 Add steps here to build the language server itself 👀
+# CMD ["echo", "🚨 This statement should be removed once you have added the logic to start up the language server! 🚨 Exiting..."]
+
+FROM node:9
+
+# Use tini as entrypoint to correctly handle signals and zombie processes.
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+RUN chmod +x /tini
+ENTRYPOINT ["/tini", "--"]
+
+USER node
+# see https://github.com/nodejs/docker-node/blob/master/docs/BestPractices.md#global-npm-dependencies
+ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
+ENV PATH=$PATH:/home/node/.npm-global/bin
+RUN npm i -g ocaml-language-server reason-cli@3.1.0-linux
+
+COPY --from=0 /usr/local/bin/lsp-adapter /usr/local/bin
+EXPOSE 8080
+# Modify this command to connect to the language server
```

Comment:
should we keep these comments?

Reply:
I like having them to show how we're modifying the example Dockerfile.